content/_code-samples/freeze/py/check_no_freeze.py

```diff
@@ -0,0 +1,45 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountInfo
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to testnetwork
```

Comment:
The test network resets every 45-90 days, so if you want to ensure that an account has `lsfNoFreeze` enabled you'll probably want to do it within this script, or just reference an account on mainnet which has that setting since mainnet doesn't get reset. (As it stands the account address below will disapper in a month or two)

Reply:
Do you have any mainnet account i can use

Reply:
This account has an actively frozen trustline on mainnet: https://xrpscan.com/account/rUpy3eEg8rqjqfUoLeBnZkscbKbFsKXC3v

Also here are two example transactions with freeze in the trust set- https://github.com/ripple/explorer/issues/393

Reply:
(Sorry for the delayed response - my notifications were messed up for a week)

Reply:
This also has frozen trustlines. https://xrpscan.com/account/rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn